optimera-htb.js

```diff
@@ -142,11 +152,27 @@ function OptimeraHtb(configs) {
          */
 
         /* ---------------------- PUT CODE HERE ------------------------------------ */
+        // https://s3.amazonaws.com/worthleydev/scores-json.js
+        var site = __getSite();
+
         var queryObj = {};
         var callbackId = System.generateUniqueId();
+        var host = Browser.getHostname();
+        var url = Browser.getPageUrl();
+        var path = url.replace(/^.*\/\/[^\/]+/, '');
+
+        if (path == '/') {
+          path = '';
+        }
 
         /* Change this to your bidder endpoint.*/
-        var baseUrl = Browser.getProtocol() + '//someAdapterEndpoint.com/bid';
+        var baseUrl = Browser.getProtocol()
+            + '//s3.amazonaws.com/optimera-client/'
+            + site.clientID
+            + '/'
+            + host
+            + path
+            + '.js';
```

Comment:
I noticed that on Prebid.js there is a cache buster appended to this URL. If that is needed I would recommend using `System.generateUniqueId()` to generate a random ID on load in `__constructor()` and using that for the lifetime of the page load.

This way we can ensure that we try to retrieve the file once per page load, the rest of the retrievals should be handled by the browser's cache.

If cache busting is not needed please let me know and I will resolve this comment.

Reply:
@ix-certification thanks for catching that. Yes, I added that.